packages/message/src/repositories/MessageRepository.ts

```diff
@@ -0,0 +1,52 @@
+// import { StorageFactory } from '@adorsys-gis/storage';
+import { StorageFactory } from '@adorsys-gis/storage';
+import { DBSchema } from 'idb';
+import { Message } from '../model/Message';
+
+interface MyDatabase extends DBSchema {
+  messages: {
+    key: number;
+    value: Message;
```

Comment:
Is the key of the Message model a number? Does this work?

Reply:
Yes , with the number it is working Sir. I will push again with the changes then you can check again.